packages/webdriverio/src/commands/browser/setTimeout.ts

```diff
@@ -43,19 +43,20 @@ export default async function setTimeout(timeouts) {
 
     const implicit = timeouts.implicit
     // Previously also known as `page load` with JsonWireProtocol
-    const pageLoad = timeouts['page load'] || timeouts.pageLoad
+    const pageLoad = (timeouts as any)['page load'] || timeouts.pageLoad
     const script = timeouts.script
+    const setTimeouts: any = this.setTimeouts.bind(this)
```

Comment:
should setTimeout and timeouts exist in the browser object?

Reply:
implementations between JSONWireProtocol and WebDriver are different which is impossible with the current setup to differentiate